src/components/Staff/Checkin/Failed.tsx

```diff
@@ -0,0 +1,9 @@
+const Failed = () => {
+  return (
+    <div className="mt-8 grid h-16 w-80 place-content-center rounded-2xl border-2 bg-indigo-950 font-bold">
```

Comment:
Next time, please avoid using margins in reusable components as it can make future spacing adjustments more difficult.